src/main/resources/uischema/fetchDirectoryContent.json

```diff
@@ -0,0 +1,118 @@
+              "groupName": "Basic",
+              "elements": [
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "directoryPath",
+                    "displayName": "Directory Path",
+                    "inputType": "stringOrExpression",
+                    "defaultValue": "",
+                    "required": "true",
+                    "helpTip": "The path to the directory to list files"
+                  }
+                },
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "matchingPattern",
+                    "displayName": "Matching Pattern",
+                    "inputType": "stringOrExpression",
+                    "defaultValue": "*.*",
+                    "required": "false",
+                    "helpTip": "Pattern to match when listing files"
+                  }
+                },
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "recursive",
+                    "displayName": "List Files In Sub-Directories",
+                    "inputType": "booleanOrExpression",
+                    "defaultValue": "false",
+                    "required": "false",
+                    "helpTip": "Whether to list files in sub-directories"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "type": "attributeGroup",
+            "value": {
+              "groupName": "Advance",
+              "elements": [
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "sortingAttribute",
+                    "displayName": "File Sort Attribute",
+                    "inputType": "comboOrExpression",
+                    "comboValues": ["Name", "Size", "LastModifiedTime"],
+                    "defaultValue": "Name",
+                    "required": "true",
+                    "helpTip": "Files will be listed as per attribute."
+                  }
+                },
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "sortingOrder",
+                    "displayName": "Sort Order",
+                    "inputType": "comboOrExpression",
+                    "comboValues": ["Ascending", "Descending"],
+                    "defaultValue": "Ascending",
+                    "required": "false",
+                    "helpTip": "Order you need to sort in ascending or descending order"
+                  }
+                },
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "resultPropertyName",
+                    "displayName": "Property Name",
+                    "inputType": "stringOrExpression",
+                    "defaultValue": "",
+                    "required": "false",
```

Comment:
Shall we make this as required. and put a default value.

Reply:
Fixed.